tgt_loop.c

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT or GPL-2.0-only
-
+#include <climits>
```

Comment:
I'd suggest to add '#include <limits.h>' to include/ublksrv.h, so that any target can
use it without including this header.